packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -269,3 +262,43 @@ contract OptimismPortal_Test is L2OutputOracle_Initializer {
         op.finalizeWithdrawalTransaction(0, alice, alice, 0, 0, hex"", 0, outputRootProof, hex"");
     }
 }
```

Comment:
Can we add a test for accessing the reserved `__gap` in an upgrade?

Reply:
TBH I wasn't totally sure what you were looking for, but I think this gets at it: 

[`0965fc4` (#2825)](https://github.com/ethereum-optimism/optimism/pull/2825/commits/0965fc43075da77659783dc7db7e3962e65ad92b#diff-5e974f8d325ddec55609e9fac6947a369a9376db659fb40eeaf426537b54b3fe)

Reply:
This does look like its testing what was requested